scripts/config.guess

```diff
@@ -1,4 +1,4 @@
-#! /bin/sh
+#! /bin/bash
```

Comment:
This one script is an exception: it is a very old bit of core configuration scripting and is specifically written to use `sh` features only.  As such I'd prefer to leave this unchanged.